src/libcollectionstest/lib.rs

```diff
@@ -15,6 +15,7 @@
 #![feature(collections_drain)]
 #![feature(core)]
 #![feature(const_fn)]
+#![feature(extend_ref)]
```

Comment:
Did the compiler tell you you needed to add this attribute? (or was it just preemptively added?)